terms.html

```diff
@@ -3,14 +3,23 @@
 </p>
 
 <dl class="termlist">
+  <dt><dfn data-lt="author|authors|authored|author's|author(s)|authoring">author</dfn></dt>
+  <dd>
+A role that a <a>party</a> performs as it expresses an assertion about a particular <a>entity</a>, i.e., creates a <a>claim</a>.
+  </dd>
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>party</a>.
+The entity is called the <a>subject</a> of the claim. 
+The <a>party</a> is referred to as the <a>author</a> of the claim, i.e. the one that has expressed the claim.
+The meaning of the claim (its semantics) is determined by its <a>author</a>.
+Different <a>authors</a> can reuse the same semantics for claims through the
+use of author-independent, shared vocabularies.
   </dd>
-  <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
+    <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
   <dd>
-A set of one or more <a>claims</a> made by an <a>issuer</a>. 
-The <a>claims</a> in a credential can be about different <a>subjects</a>.
+A set of one or more <a>claims</a> <a>authored</a> by a single <a>party</a>, that is constructed for the purpose of transferring it to some other <a>party</a>. 
```

Comment:
Why does an implementer need to understand the difference between an `author` and an `issuer`?

Reply:
An implementer may not need to understand the difference between an author and an issuer. However, in my opinion, standards such as these should not limit their audience to implementers. They should also enable non-implementers to know precisely what they're talking about.

I am not an implementer, but I do contribute to architectures, systems designs, etc. I need to know what I'm talking about to help projects make the right decisions. Texts that 'sort of' say what VCs are do not help me. I need texts, which I hope that standards are, that precisely state the facts of the matter, in order to avoind making mistakes. That particular characteristic determines the effectiveness of specifications as a vehicle for briding the gap between implementers and non-implementers.

Regarding the terms 'author' and 'issuer': as @dlongley pointed out, there is no such thing as the issuer of a claim, because they can exist without there being a credential that contains them. Having 'author' for a claim allows us to talk about the source of the claim. This is also helpful when talking about claims that are part of presentations. It makes it easier (for non-implementers) to understand the necessity of having proofs of authorship of claims therein. Also, it makes it easier to explain (to non-implementers) what the difference is between credentials (a (signed) set of claims, where the issuer is the author of each of them) and presentations (a (signed) set of claims, where the claims can have different authors and there is no implied link between the party that created (and signed) the presentation and the authorship of the claims).

Reply:
> Regarding the terms 'author' and 'issuer': as (I think) @dlongley pointed out, there is no such thing as the issuer of a claim, because they can exist without there being a credential that contains them. Having 'author' for a claim allows us to talk about the source of the claim.

I'm curious where @dlongley pointed out that there is no such this as an issuer of a claim. I would argue that such a distinction is a distinction without a difference.

I would argue that the issuer is the author. 

Full stop.

Signing the claim means that the authority who is signing it is attesting to that truth of the claims. They are the author of those claims in that they are making that statement and attesting to their veracity. Whether or not they are the only author or the original author is irrelevant.

You *could* define an author that is different from the issuer, but that would have no impact on the meaning of the VC. Perhaps if the VC is itself a statement of authorship by someone other than the issuer of the VC. 

In that case, you're just adding another claim asserted by the issuer, which verifiers will interpret in some way. That is, a VC issued by BobCo (using a DID we believe is associated with BobCo for attestations) could say "AuthorDan wrote that the sky is blue", i.e., "AuthorDan is the author of the claim that the sky is blue"

However, the VC can only validate that BobCo says "AuthorDan is the author of the claim that the sky is blue".  Whether or not AuthorDan is, in fact, the author of those claims is opaque to the VC data model. 

In contrast, the fact that the DID known to be associated with BobCo has an attestationMethod that can cryptographically verify the message is not tampered with is concrete evidence that the statement is made/authored/attested by BobCo.

In short, the only meaning of "author" that applies to a VC is the issuer. While that doesn't preclude the issuer making claims about authorship, we do not have any mechanism to deal with "author" of a VC as anything other than the Issuer.

Reply:
You're right. I thought I remembered what @dlongley said, but didn't. 

I agree with you that its author is its issuer, but *only* in the context of VCs. 

In the context of VPs, it's another matter. It is possible that a VP contains claims that are authored by the party that performs the role of the holder that creates the VP. Since such claims were never part of a VC, there is no issuer for them, but there is authorship. In fact, the very definition of 'verifiable presentation' talks about authorship, saying: "A verifiable presentation is a tamper-evident presentation encoded in such a way that *authorship of the data* can be trusted after a process of cryptographic verification."

Reply:
@jandrieu, @OR13: given the explanations above, is there anything concrete that you would like me to address so that the PR can proceed?

Reply:
with VPs, I would argue that when a holder creates a VP which includes claims that were never part of a VC, they are acting as an issuer.

The addition of 'author' and the attempt to conceptually differentiate it from 'issuer' results in increased confusion for me. I read through the proposed changes and wrap my head around the two concepts, then am left to wonder why the distinction is being made at all. 

Rather than adding terms to the terminology section, I would much prefer we simplify it by removing terms.

Reply:
@brentzundel: I cannot understand your argument because the terms you use do not seem to mean what the VCDM says they do. The term 'issuer', for example, is defined by the VCDM specifically in relation to the creation of a VC. So, your thinking that a holder could create a VP that includes claims _that were never part of a VC_ is inconsistent with how VCDM defines 'issuer, so such a holder would not be acting as an issuer. 

This may seem nit-picking, but it is an example of how many discussions/issues that we have get lengthy and aren't resolved in a satisfactory way. It would help so much if we could get ourselves to use terms that are defined *only* in the way in which they are defined, which is the start of having/creating a single understanding of what it is we're talking about, that we then all all share/commit to use.

The current set of terms does not allow us to properly describe the situation you refer to, in which a holder creates a VP which includes claims that were never part of a VC. This is a relevant use-case. In order to be able to talk about this, we might, e.g., modify the term 'issuer' to accommodate for that, introduce 4 roles rather than the current 3 by splitting up the 'holder' role, and I wouldn't be surprised if there are more ways to do this. I remember various discussions/issues about this which didn't lead anywhere.

The term 'author' is really nothing very special. It is just a generic way to refer to the party that created some data (and might be held accountable for its truth, trustworthiness, or the fact that it has created it). It is a simple concept that can be used to refer to the party that has created an individual claim, VC, VP, presentation request, etc. The roles of issuer, holder and verifier are specializations, where an issuer authors VCs for the purpose of sending them to a holder, a holder authors VPs (that includes VCs and/or claims authored by itself or other parties), and a verifier authors presentation requests (in which it seeks to obtain VPs that contain VCs and/or claims that are authored by particular parties). The term 'author' makes it much simpler to say these things than if I had to use the terminology as currently defined, and it is also easier to understand for those that use VCDM terminology rather than their own).

Reply:
@RieksJ,

My understanding is that some people are objecting to this PR because `author` is being introduced as a separate and explicit role at the same level of `issuer`, `holder`, and `verifier`. I wonder if you could make the clarifications in terminology you'd like to see by not making it an explicit separate role, but by still talking about the existing roles performing `authoring` actions, etc. This would be my suggestion to try and move this PR toward consensus.

Reply:
Thanks @dlongley for the suggestion. I am confused about people objecting to a term they also use in daily life. In the context of books, there is an author, and you have publishers (issuers if you will), stores, libraries and other roles that can be performed and from where you can request for and obtain books. People tend to be interested to learn who the author of a book is, and not so much the path that the book went to end up with the reader/user. I have made some changes to the definition to clarify this aspect.

Reply:
the re-wording of author makes me dislike it less.

Reply:
@brentzundel That sounds as an improvement. Unless there are concrete suggestions for changes, I guess it is acceptable.